src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -1090,7 +1090,7 @@ public async Task TestMemberOfStructFromSource()
 static class Test { int a = MyStruct.Some$$Field; }";
 
             await TestAsync(markup,
-                MainDescription($"({FeaturesResources.field}) int MyStruct.SomeField"));
+                MainDescription($"(static {FeaturesResources.field}) int MyStruct.SomeField"));
```

Comment:
this is a bit strange.  we are mixing localized and non-localized strings here.

Reply:
This may not be an acceptable way to do things across different langauges.

Reply:
Given that `static` is a keyword, would it ever make sense to localize it?

Reply:
Given that the tests in this file are C#-specific, I'm not sure if we need to worry about how other languages will handle this in this particular code file... Perhaps I'm misunderstanding

Reply:
My 'localization' i mean things like French/English, etc.  for example, imagine a language that would want to say ```(Field, static)``` not ```(static Field)```.  Perhaps just say ```(Field) static int Whatever```?

Reply:
I was trying to following the design given in issue [#26947](https://github.com/dotnet/roslyn/issues/26947#issuecomment-390831445), which calls out the desire to have the keywords placed in the parentheses (e.g. `(volatile field)` or `(static readonly field)`

Reply:
Right.  The design may be fine, but htis would not be the right way to code it up.  Specifically, the act of manually concatting words, and manually placing spaces is not something that localizes properly.  To do this right, there needs to be a localized resource (like `({0} field)`) that is used here where the modifiers are substituted into that location.  That way the localization people can decide the right way for this to work in any differnet language.

Reply:
I see, I think I understand your concerns much better now. Thank you. So I guess my concern with that approach would be this:

Currently, it's coded such that the keywords (`static`, `readonly`, `volatile`) have the syntax highlighting (blue font by default I believe) in the tooltip. The "field" word does not; it's just plain text. This is possible because the keywords and the "field" word are separate `SymbolDisplayPart`s. Adding localization like that would force us to combine the keywords and field word into one single block of text before they are added to the tool tip.

Should that single combined block of text just be entirely plain text and forget about syntax highlighting the keywords? Or should we attempt to parse out the "field" word from the localized resource as plain text and syntax highlight the remaining words?

Reply:
> Should that single combined block of text just be entirely plain text and forget about syntax highlighting the keywords? Or should we attempt to parse out the "field" word from the localized resource as plain text and syntax highlight the remaining words?

Basically, hearing this makes me feel that this is super weird to begin with :)   Why are we mixing 'human' words with 'programming language keywords' to try to form some sort of 'phrase'.  It's not really something sensible to me.  We already have a section for the 'programming language portion'.  i.e. *today* we have something like `(Field) int _myField;`.  The `(...)` is the human-language portion.  The `int _myField` is the programming language portion.  

So why not keep both those sections as they are today?  With this new work, that would mean getting a display like `(Field) static readonly int s_mySharedData;`.  Now, nothing is strange about 'Field'.  It is a human-language term, and it continues to localize to foreign languages just fine.  And we have the section for the actual programming language code where it makes sense to colorize keywords, type names, and whatnot.  

So that would *definitely* be my preferred approach here.

--

Also, given that thsi is a change in UI, it would be great if screen snippets were provided in the bug.  Seeing the colorized '(static field)' portion would likely have been a strong signal that something was a bit weird here.

Thanks! :)

Reply:
IMO this is actually a weird design to mix language keyword with an (english) description name as in (static field). I would expect static to be be colored as a keyword and appear in the declaration, outside parentheses.

Reply:
Descriptions, which are localizable, should be in parentheses. Language keywords and tokens should not in my view.

Reply:
I cannot even imagine how this would be localized in my native language for example. You can't easily mix (english) language keywords and the description in one phrase.

Reply:
@sharwell, do you have an opinion on this? There seems to be a consensus that inserting the keywords outside the parentheses makes more sense linguistically and logically than inserting them inside the parentheses, but that deviates from the design in [#26947 (comment)](https://github.com/dotnet/roslyn/issues/26947#issuecomment-390831445). Is it alright to make that change, or are we committed to the design as it is now?

Reply:
Note: @Neme12 's concerns are very relevant here.  It's good to have a non-native language speaker to weigh in here.  While certain approaches may feel right in english, they can easily break down in many other languages.

Reply:
I can't speak for other languages but if we did this sort of this in my language - combining `static` and a translation of field in a single phrase, depending on the order, it would either sound like incorrect grammar or like `static` is the name of the field. If we want to use an adjective-noun phrase here, then both words should be localized (to make a proper phrase in non-english languages), otherwise the word `static` should be outside the parantheses.

Reply:
I updated the code and original post to include the proposed changes (keywords after the "(field)" text). It should be easy enough to revert if necessary